tests/simple_vb_with_correlation_analytical.py

```diff
@@ -108,8 +108,8 @@ def main(_plot=True):
     
     plot_posteriors([info, info2, info3], ['Without correlation', 'With target correlation', 'Analytical'])
     
-    err_mean = abs( (info2['mean'] - info3['mean']) / info2['mean'])
-    err_precision = abs( (info2['precision'] - info3['precision']) / info2['precision'])
+    err_mean = abs(info2['mean'] - info3['mean']) 
```

Comment:
As long as we are sure that we do not have ZERO values (which is the case, here), I would be more confident with relative error than absolute error, since it is IMO a more fair measure for the comparison. So, I did not quite get the point/necessity of using absolute error.

Reply:
We want the error to be numerically zero, so I'd say that it makes no difference. And in this case, I always go for less code.